src/app/dashboard/dashboard.scss

```diff
@@ -0,0 +1,46 @@
+#userbar {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 46px;
+	background-color: #7d8fb3;
+	padding: 5px;
+	margin-bottom: 10px;
+}
+
+.ticker {
+	display: block;
+	width: 240px;
+	height: 112px;
+	padding: 5px;
+	background-color: #AAA;
+	color: #FFF;
+	font-family: Helvetica, Arial, Sans-serif ;
+	margin:10px ;
+	border-radius:5px;
+}
+
+.ticker * {
+	padding: 2px;
+	margin:0px;
+}
+
+.ticker span {
+	width: 10px;
+	height: 10px;
+	border-radius: 4px;
+	background-color: red;
+	display: block;
+	margin: 3px ;
+}
+
+.ticker h2{
+	text-align: right;
+}
+
```

Comment:
I will have to teach you sass next week.  everything with .ticker can be nested: